### DISCUSSION TITLE
v3版播放按钮potplayer的链接有问题，无法播放

### DISCUSSION THREAD

**Discussion by iswazy** [AlistGo/alist#2797] [Q&A]
用的是PC测试，启动播发器后提示找不到文件。使用其他播放器按钮提取的链接直接在potplayer里面进行播放是可以的。可以判断potplayer按钮提取的播放链接存在问题。

**Comment by Faxmell** [comment]
似乎是最近刚有的这个问题，我也出现了，利用webdev挂载可以看，但是在alist启动potplayer就会出现这种情况

**Comment by anwen-anyi** [comment]
https://github.com/alist-org/alist/issues/2782 貌似修复了等下个版本试试看
https://github.com/alist-org/alist-web/commit/d95c5d526bb43496e1664eca74c13df4489486ce

**Reply by viryaka** [reply]
我的也是调不起potplayer，等新版本
